osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-smr43.json

```diff
@@ -187,6 +187,54 @@
           "access": "R"
         }
       ]
+    },
+    {
+      "tag": "ALARM_REGISTER_1",
+      "description": "Alarm register 1",
+      "note": null,
+      "class-id": 1,
+      "version": 0,
+      "obis": "0.0.97.98.0.255",
+      "group": "ELECTRICITY",
+      "meterTypes": [
+        "SP",
+        "PP"
+      ],
+      "properties": {},
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "double-long-unsigned",
+          "valuetype": "DYNAMIC",
+          "value": "0",
+          "access": "RW"
+        }
+      ]
+    },
+    {
+      "tag": "ALARM_FILTER_1",
```

Comment:
Waarom staat ALARM_FILTER_1 hier, deze is identiek aan 4.2.2.
De spec van 4.3 overerft alle specificaties van 4.2.2

Reply:
Overerving had ik niet goed in de gaten, dit is nu aangepast.